packages/amplify-codegen/package.json

```diff
@@ -23,7 +23,7 @@
     "@aws-amplify/appsync-modelgen-plugin": "1.32.0",
     "@aws-amplify/graphql-docs-generator": "2.4.2",
     "@aws-amplify/graphql-types-generator": "2.8.6",
-    "@graphql-codegen/core": "1.8.3",
+    "@graphql-codegen/core": "2.5.1",
```

Comment:
I think the graphql-codegen/core dep bump might actually be what caused some issues in studio testing a few weeks back. Is this required for the upgrade?

Reply:
@alharris-at  The older version of graphql-codegen/core pull in an older version of lo-dash which has two dependabot warnings 
1.  high          │ Command Injection in lodash  │ Patched in    │ >=4.17.21                                                    │

    │ @aws-amplify/cli-internal > amplify-container-hosting >      │
    │ @aws-amplify/amplify-category-api >                          │
    │ amplify-provider-awscloudformation > amplify-codegen >       │
    │ @graphql-codegen/core > @graphql-codegen/plugin-helpers >    │
    │ @graphql-toolkit/common > lodash                             │
 More info     │ https://www.npmjs.com/advisories/1068132

Reply:
We had to revert this PR previously https://github.com/aws-amplify/amplify-codegen/pull/325/files due to studio impact. Are we able to just bump the lodash dep as an override or something along those lines?

Reply:
I suspect if we move forward with this change we'll fail studio smoke tests during the CLI release process, so just trying to get ahead of that.

Reply:
What is the minimum version of lodash needed to fix the vulnerability?

Reply:
The mininum version is 4.17.21. We do not have direct dependency but a downstream dependency of `graphql-codegen` packages

Reply:
I tried changing `@graphql-codegen/core` to `1.17.10` to avoid the breaking change. It appears that this version fixes the lodash dependency issue. 

```
> yarn list lodash 
yarn list v1.22.18
warning Filtering by arguments is deprecated. Please use the pattern option instead.
└─ lodash@4.17.21
```

Reply:
That might fix but still we do not know the impact of the version `1.17.10` to studio. Do we have any context from last failure for studio? We can analyze the root cause and figure out a solution to avoid the failure.

Reply:
@AaronZyLee could we please test this with Studio E2E in a tagged release.

Reply:
@sachscode Sure. I will rebase this PR to a tag release branch and run codegen tag release from there.